druntime/src/importc.h

```diff
@@ -57,6 +57,9 @@
 #define __int32 int
 #define __int64 long long
 
+/* Digital Mars C */
+#define __imaginary 1.0i
```

Comment:
how is `__imaginary` used? This seems like it should be 
```c
#define __imaginary 1.0i *
```
or 
```c
#define __imaginary  * 1.0i
```

Reply:
`__imaginary` is treated as 1.0i by dmc.